examples/spark-with-S3/Dockerfile

```diff
@@ -0,0 +1,55 @@
+# docker file/docker-compose derived from https://github.com/gettyimages/docker-spark
+# https://github.com/gettyimages/docker-spark/blob/master/LICENSE
+FROM debian:stretch
+
+RUN apt-get update
+#    && apt-get install -y curl unzip \
+#    && apt-get install -y python-pkg-resources python3.6 python3.6-venv python3-venv
```

Comment:
If they are commented can we remove these ?

Reply:
Done